build/ci/evergreen.yml

```diff
@@ -2043,6 +2067,9 @@ patch_aliases:
   - alias: "cleanup"
     variant_tags: [".cleanup .cron"]
     task: ".*"
+  - alias: "snyk"
+    variant_tags: [".snyk .cron"]
+    task: ".*"
 github_pr_aliases:
```

Comment:
I did not add Snyk here because the current command scans + uploads the results.
We don't want PR results to be uploaded.